public/index.css

```diff
@@ -1,74 +0,0 @@
-@import url("https://fonts.googleapis.com/css2?family=Quicksand:wght@400;500;700&display=swap");
-
-* {
-  font-family: "Quicksand";
-  border-radius: 8px;
-}
-
-#homeTitle {
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
-  background-color: peachpuff;
-}
-
-h1 {
-  font-size: 36pt;
-  text-align: center;
-  font-weight: 700;
-  color: rgb(87, 29, 7);
-}
-
-#welcome {
-  text-align: center;
-}
-
-h3 {
-  font-weight: 700;
-  margin-bottom: 0px;
-  margin-right: 5px;
-  text-decoration: underline;
-  color: rgb(151, 51, 14);
-}
-
-a {
-  margin-bottom: 32px;
-  margin-right: 5px;
-  font-weight: 500;
-  color: orangered;
-  text-decoration: none;
-}
-
-a:hover {
-  color: rgb(177, 59, 17);
-}
-
-a:visited {
-  color: rgb(97, 187, 243);
-}
-
-#mapContainer {
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
-  margin-top: 10px;
-}
-
-#mapNav {
-  display: flex;
-  flex-direction: column;
-  width: 200px;
-  text-align: right;
-  background-color: rgb(245, 232, 222);
-}
-
-#mapid {
-  height: 400px;
-  width: 600px;
-  margin-left: 5px;
-  border: orange solid 2px;
-}
-
-#restPage {
-  font-size: 24pt;
-}
```

Comment:
You have great commenting and notes everywhere, except here. Even if its just organizational comments such as sidebar, header, footer, map etc, that is preferred.